lib/system.nim

```diff
@@ -54,6 +54,16 @@ type
 
 include "system/basic_types"
 
+const
+  NimMajor* {.intdefine.}: int = 1
+    ## is the major number of Nim's version.
+
+  NimMinor* {.intdefine.}: int = 3
+    ## is the minor number of Nim's version.
+    # Odd for devel, even for releases.
```

Comment:
Who is this comment for?

Reply:
For anyone looking at the code (just like any regular non doc comment).

Reply:
turned it into a regular doc comment